cardano-cli/src/Cardano/CLI/Byron/Delegation.hs

```diff
@@ -20,7 +20,7 @@ import           Cardano.Config.Types (CertificateFile (..))
 import qualified Cardano.Crypto as Crypto
 import           Cardano.Crypto (ProtocolMagicId)
 
-import           Cardano.CLI.Ops
+import           Cardano.CLI.Errors (CliError(..))
```

Comment:
:100: This has been annoying me for the longest time but was worried about breaking WIP by moving it.